js/values/Paper.js

```diff
@@ -21,7 +21,7 @@ var defaultAttributes = {
  * @returns {object} The attributes passed to the Paper.
  */
 var sanitizeAttributes = function( attributes ) {
-	attributes.keyword = sanitizeString( attributes.keyword );
+	attributes.keyword = escapeRegExp( attributes.keyword );
```

Comment:
Perhaps change method name if you change the action in the function is changed to escape instead of sanitize